cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,139 @@
+
+Even when our internal developers cannot work on your issue, that does not mean all hope is lost!  Being an open source project means you can be part of the solution. We love when our community commits code and want to encourage everyone to feel empowered to contribute and make the product we all love and use even better.  We have guides on [how to contribute](https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md) and a very active [Discord community](https://discord.gg/cypress) which can help if you are interested in opening a PR.  
+
+### Why did you close my ticket?
+
+There are a number of reasons why a ticket may be closed without any change or PR being opened.
+
+- **No Response From Author** -   The most common reason is lack of response from the author. Our issue creation template prompts the user for many details that are vital in debugging and replicating an issue. It is not uncommon for issues to be entered with insufficient information for our teams to properly investigate an issue. We will often reach out for more details, but we do not have the bandwidth to chase down users for information. **If we do not receive a response within 7 days, we will close your ticket.** The best way to help get your issue worked on by a Cypress team member is to provide the information requested and give as much detail as possible (or, even better, a reproducible example in our [cypress-tiny repo](https://github.com/cypress-io/cypress-test-tiny)) in a timely manner.
+
+- **Not a bug or feature request** - Issues entered into the [Cypress repo](https://github.com/cypress-io/cypress) are for bugs and feature requests for the Cypress App only. Updates to [documentation](https://github.com/cypress-io/cypress-documentation), our [example-kitchensink app](https://github.com/cypress-io/cypress-example-kitchensink), or [another repository](https://github.com/cypress-io) should be made in the appropriate repository. 
+The best place for asking questions is our [Discord server](https://discord.gg/cypress) which has a very active community of folks with a diverse set of knowledge. Other available channels to explore include [Cypress GitHub discussions](https://github.com/cypress-io/cypress/discussions), [community chat](https://on.cypress.io/chat), and [Stack Overflow](https://stackoverflow.com/questions/tagged/cypress).
+We also offer support via email with our [paid plans](https://www.cypress.io/pricing/).
+
+- **Feature request for Cypress Dashboard** - Thank you for your support as a dashboard user! These issues are routed to our dashboard team's ticketing system. Your customer success representative is available for follow-up and will reach out you directly via email if more information is needed.
```

Comment:
Haa it's confusing but we do capitalize Dashboard, just not app

Reply:
updated